### PR TITLE
Upgrade to Scala.js 1.0.0-RC2

### PR DESCRIPTION
**Pull request by lolgab**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -38,6 +38,6 @@ before_cache:
   - find $HOME/.ivy2/cache -name "ivydata-*.properties" | xargs rm
 cache:
   directories:
-  - $HOME/.ivy2/cache
-  - $HOME/.sbt/boot
-  - $HOME/.sbt/launchers
+    - $HOME/.ivy2/cache
+    - $HOME/.sbt/boot
+    - $HOME/.sbt/launchers
```
Comment (sjrd) [inline .travis.yml]: This change is unrelated. Please revert it and amend your commit.

**Review by sjrd — CHANGES_REQUESTED** [review]

**Comment by sjrd** [comment]
Note that portable-scala-reflect 0.1.1 is already published for Scala.js 1.0.0-RC2. You can already use it today.

File: .travis.yml (hunk 1 above)
Comment (lolgab) [reply .travis.yml]: Sorry, I have a linter running on save.
✅Done

**Comment by lolgab** [comment]
> Note that portable-scala-reflect 0.1.1 is already published for Scala.js 1.0.0-RC2. You can already use it today.

Didn't know that. Thank you

**Review by sjrd — APPROVED** [review]
Thanks.

**Comment by sjrd** [comment]
> > Note that portable-scala-reflect 0.1.1 is already published for Scala.js 1.0.0-RC2. You can already use it today.
> 
> Didn't know that. Thank you

In general, anything in the scala-js and portable-scala organizations, you can assume that the latest release supports the latest Scala.js. They are all under my control and have no external dependencies, so I always make them available.
